packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -556,9 +570,15 @@ mixin WidgetsBinding on BindingBase, SchedulerBinding, GestureBinding, RendererB
   /// Whether the Flutter engine has rasterized the first frame.
   ///
   /// {@macro flutter.frame_rasterized_vs_presented}
-  Future<void> get firstFrameRasterized => _firstFrameCompleter.future;
+  bool get firstFrameRasterized => _firstFrameCompleter.isCompleted;
 
-  /// Whether the first frame has finished rendering.
+  /// A future that completes when the Flutter engine has rasterized the first
+  /// frame.
+  ///
+  /// {@macro flutter.frame_rasterize`_vs_presented}
+  Future<void> get waitUntilFirstFrameRasterized => _firstFrameCompleter.future;
```

Comment:
The name is a little misleading because calling the getter doesn't actually wait for anything. The caller has to do the waiting by awaiting on the result. Not sure what a better name would be, though...

Reply:
I can't think of a better name either. But since Dart never has a synchronous waiting mechanism, it seems to be Ok to return a Future for a `waitUntilXYZ` function? The only way to wait in Dart is `await` (see, e.g., https://github.com/flutter/flutter/blob/master/packages/flutter_driver/lib/src/extension/extension.dart#L242)?